cypress/e2e/tests/migration/dynamic-reports/filter.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Igor , I think we should name this folder also as "Issues" otherwise there might be confusion about corresponding model for this test .